### PR TITLE
Adding unit tests for 'app/process/handler/read-only/nft/fee/handler'.

### LINKED ISSUE #445: Unit tests for app/process/handler/read-only/nft/fee/handler.go
- Implement unit tests for **app/process/handler/read-only/nft/fee/handler.go**

### PR DESCRIPTION
**Pull request by Psykepro**
Signed-off-by: Nikolay Nedkov <nikolay.nedkov@limechain.tech>

**Detailed description**:
Adding unit tests for 'app/process/handler/read-only/nft/fee/handler'.

**Which issue(s) this PR fixes**:
Fixes #445 

**Special notes for your reviewer**:

**Checklist**
- [ ] Documentation added
- [x] Tests updated

### CODE REVIEW

**Review by svetlio8 — APPROVED** [review]

**Review by vectorman1 — APPROVED** [review]

File: app/process/handler/read-only/nft/fee/handler.go [hunk 1]
```diff
@@ -112,40 +112,47 @@ func (fmh Handler) Handle(payload interface{}) {
 
 	for _, splitTransfer := range splitTransfers {
 		feeAmount := -splitTransfer[len(splitTransfer)-1].Amount
-
 		fmh.readOnlyService.FindAssetTransfer(transferMsg.TransactionId, constants.Hbar, splitTransfer,
 			func() (*mirror_node.Response, error) {
-				return fmh.mirrorNode.GetAccountDebitTransactionsAfterTimestampString(fmh.bridgeAccount, transferMsg.Timestamp)
+				return fmh.fetchHandler(transferMsg)
```
Comment (failfmi) [inline app/process/handler/read-only/nft/fee/handler.go:117]: `fetchHandler` -> `fetch`

File: test/constants/bridge.go [hunk 2]
```diff
@@ -112,9 +112,9 @@ var (
 		constants.EvmDefaultDecimals,
 	}
 
-	/////////////////////
+	// ///////////////////
 	// Polygon Network //
-	/////////////////////
+	// ///////////////////
 
 	// Native Tokens //
```
Comment (failfmi) [inline test/constants/bridge.go:120]: Are these changes necessary? Check above as well.

File: app/process/handler/read-only/nft/fee/handler.go [hunk 3]
```diff
@@ -112,40 +112,47 @@ func (fmh Handler) Handle(payload interface{}) {
...
-					TransferID: sql.NullString{
-						String: transferMsg.TransactionId,
-						Valid:  true,
-					},
-				})
-				if err != nil {
-					fmh.logger.Errorf("[%s] - Failed to create fee  entity [%s]. Error: [%s]", transferMsg.TransactionId, scheduleID, err)
-				}
-				return err
+				return fmh.saveHandler(transactionID, scheduleID, status, transferMsg, feeAmount)
```
Comment (failfmi) [inline app/process/handler/read-only/nft/fee/handler.go:120]: `saveHandler` -> `save`

**Review by svetlio8 — APPROVED** [review]

**Review by failfmi — APPROVED** [review]
